examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -367,6 +399,19 @@ def main():
 
     config.update({"forced_decoder_ids": model_args.forced_decoder_ids, "suppress_tokens": model_args.suppress_tokens})
 
+    if config.model_type == "whisper":
+        config.update(
+            {
+                "apply_spec_augment": model_args.apply_spec_augment,
+                "mask_time_prob": model_args.mask_time_prob,
+                "mask_time_length": model_args.mask_time_length,
+                "mask_time_min_masks": model_args.mask_time_min_masks,
+                "mask_feature_prob": model_args.mask_feature_prob,
+                "mask_feature_length": model_args.mask_feature_length,
+                "mask_feature_min_masks": model_args.mask_feature_min_masks,
```

Comment:
I'd remove those to put good defaults and only do the update when `model_args.apply_spec_augment is True`. Then ut a comment telling the user to change the values to what they want.